tests/guilds/editGuildMfaLevel.test.ts

```diff
@@ -0,0 +1,12 @@
+import { MfaLevels } from "../../mod.ts";
+import { assertEquals } from "../deps.ts";
+import { loadBot } from "../mod.ts";
+import { CACHED_COMMUNITY_GUILD_ID } from "../utils.ts";
+
+Deno.test("[guild] edit guild mfa level", () => {
+  const bot = loadBot();
+  let elevated = bot.helpers.editGuildMfaLevel(CACHED_COMMUNITY_GUILD_ID, MfaLevels.Elevated, "test");
+  assertEquals(elevated, MfaLevels.Elevated);
+  const none = bot.helpers.editGuildMfaLevel(CACHED_COMMUNITY_GUILD_ID, MfaLevels.None, "revert test");
```

Comment:
Are there only two types, if there are more id copy paste for each one if u can

Reply:
there's only elevated and none